site/src/components/BlockCard.tsx

```diff
@@ -80,12 +80,12 @@ export const BlockCard: VFC<BlockCardProps> = ({ loading, data }) => {
     author,
     version,
     lastUpdated,
-    packagePath,
+    slug,
     icon,
   } = data;
 
   return (
-    <Link href={`/${packagePath}`}>
+    <Link href={`/${slug}`}>
```

Comment:
I think calling this `slug` is a bit confusing as it's a whole path, not a single slug - can we call it `blockPagePath` or something instead? It makes it easier to see what's being linked to without having to go look up what `slug` this is.

If you add the `/` at the point of constructing it you could pass it directly rather than interpolating everywhere as well.